### PR TITLE
Lab 5

### PR DESCRIPTION
**Pull request by CharNoir**

### CODE REVIEW

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 1]
```diff
@@ -14,6 +14,8 @@ public class TorpedoStore {
 
   private int torpedoCount = 0;
 
+  private Random generator = new Random();
```
Comment (CharNoir) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:17]: too stupid

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 2]
```diff
@@ -30,18 +32,17 @@ public TorpedoStore(int numberOfTorpedos){
...
     boolean success = false;
 
     // simulate random overheating of the launcher bay which prevents firing
-    Random generator = new Random();
     double r = generator.nextDouble();
 
     if (r >= FAILURE_RATE) {
       // successful firing
-      this.torpedoCount =- numberOfTorpedos;
+      this.torpedoCount -= numberOfTorpedos;
```
Comment (CharNoir) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:45]: FAILURE

**Comment by almatn** [comment]
nice

**Comment by MeanDiamand** [comment]
Fake Assignee

**Pull request merged**